TROUBLESHOOTING.md

```diff
@@ -0,0 +1,22 @@
+# dotnet .NET 9 SDK
+
+- Q: `dotnet restore` fails with missing NuGet packages errors
+
+- A: You need to add NuGet package source and install workload for .NET Aspire
+
+```bash
+dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org
+dotnet workload install aspire
+```
+
+- Q: running .NET Aspire project AppHost leads to errors:
+
+```text
+Container runtime 'docker' was found but appears to be unhealthy. Ensure that Docker is running and that the Docker daemon is accessible.
+```
+
+- A: You need to download Docker Desktop and install it or run it manually.

```

Comment:
Podman can be used also:  https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/setup-tooling?tabs=windows&pivots=visual-studio#container-runtime

We don't need to include all of this information, but let's defer to the official .NET Aspire docs for directions to install

Reply:
@csharpfritz I am going to remove outdated QA section and reference people to your link since on a fresh install error seems does not appear, wow.